packages/rocketchat-ui-master/client/main.js

```diff
@@ -147,6 +147,9 @@ Template.main.helpers({
 		});
 		return ready;
 	},
+	serverFirstAccess() {
+		return RocketChat.settings.get('Server_First_Access');
```

Comment:
we could test if we dont have an admin user

Reply:
We can't because the admin user will be registered one step before the latest.

Reply:
So, then how is this going to work for existing installations?